Orange/data/io.py

```diff
@@ -617,3 +631,24 @@ def read_file(self, file, cls=None):
         self.read_data(worksheet, table)
         self.reorder_values(table)
         return table
+
+
+class PickleFormat:
+    NAME = "Pickled table"
+    EXT = ".pickle"
+
+    @staticmethod
+    def read_file(file, _=None):
+        return pickle.load(open(file, "rb"))
+
+    @staticmethod
+    def write_file(filename, table):
+        pickle.dump(table, open(filename, "wb"))
+
+
+FILE_FORMATS = (TabDelimFormat, TxtFormat, BasketFormat, ExcelFormat,
```

Comment:
I would prefer a class decorator for this, so the registration of the format is closer  to the class definition.